src/Analyzer.JsonRuleEngine.UnitTests/JsonRuleEvaluationTests.cs

```diff
@@ -0,0 +1,160 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Azure.Templates.Analyzer.RuleEngines.JsonEngine.Schemas;
+using Microsoft.Azure.Templates.Analyzer.Types;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+
+namespace Microsoft.Azure.Templates.Analyzer.RuleEngines.JsonEngine.UnitTests
+{
+    [TestClass]
+    public class JsonRuleEvaluationTests
+    {
+
+        [TestMethod]
+        public void GetRuleName_ReturnsNameFromRule()
+        {
+            Assert.AreEqual(
+                "testRule",
+                new JsonRuleEvaluation(true, results: null)
+                {
+                    RuleDefinition = new RuleDefinition { Name = "testRule" }
+                }
+                .RuleName);
+        }
+
+        [TestMethod]
+        public void GetRuleDescription_ReturnsDescriptionFromRule()
+        {
+            Assert.AreEqual(
+                "test rule",
+                new JsonRuleEvaluation(true, results: null)
+                {
+                    RuleDefinition = new RuleDefinition { Description = "test rule" }
+                }
+                .RuleDescription);
+        }
+
+        [TestMethod]
+        public void GetRecommendation_ReturnsRecommendationFromRule()
+        {
+            Assert.AreEqual(
+                "test recommendation",
+                new JsonRuleEvaluation(true, results: null)
+                {
+                    RuleDefinition = new RuleDefinition { Recommendation = "test recommendation" }
+                }
+                .Recommendation);
+        }
+
+        [TestMethod]
+        public void GetHelpUri_ReturnsHelpUriFromRule()
+        {
+            Assert.AreEqual(
+                "https://helpUri",
+                new JsonRuleEvaluation(true, results: null)
+                {
+                    RuleDefinition = new RuleDefinition { HelpUri = "https://helpUri" }
+                }
+                .HelpUri);
+        }
+
+        [DataTestMethod]
+        [DataRow(3, 3, 0, DisplayName = "3/3 results are true, 0/3 results are false")]
+        [DataRow(3, 2, 1, DisplayName = "2/3 results are true, 1/3 results are false")]
+        [DataRow(3, 0, 3, DisplayName = "0/3 results are true, 3/3 results are false")]
+        [DataRow(0, 0, 0, DisplayName = "No results")]
+        public void GetResultsEvaluatedTrueFalse_ThreeResults_ExpectedNumberOfResultsReturned(int totalResults, int expectedTrue, int expectedFalse)
```

Comment:
Remove parameter `expectedFalse` and just do `totalResults - expectedTrue` in the test.

Reply:
I would also suggest simplifying the display names - 3/3 true, 0/3 false is a little redundant.